tests/cases/fourslash/codeFixUnusedIdentifier_destructure_partlyUnused.ts

```diff
@@ -0,0 +1,26 @@
+/// <reference path='fourslash.ts' />
+
+// @noUnusedLocals: true
+
+////{
+////    const { x, y } = o;
+////    x;
+////}
+////{
+////    const { x, y } = o;

```

Comment:
Would it be interesting to test the case where only one variable is declared and it is unused?  Or is that covered by the other PR?

Reply:
That should be covered by #23805.